src/NumericalAlgorithms/DiscontinuousGalerkin/Tags.hpp

```diff
@@ -35,6 +37,16 @@ struct Mortars : db::PrefixTag, db::SimpleTag {
   using Key = std::pair<::Direction<VolumeDim>, ::ElementId<VolumeDim>>;
   using type = std::unordered_map<Key, db::item_type<Tag>, boost::hash<Key>>;
 };
+
+/// \ingroup DataBoxTagsGroup
+/// \ingroup DiscontinuousGalerkinGroup
+/// Size of a mortar, relative to the element face.  That is, the part
+/// of the face that it covers.
+template <size_t Dim>
+struct MortarSize : db::SimpleTag {
+  static std::string name() noexcept { return "MortarSize"; }
+  using type = std::array<Spectral::MortarSize, Dim>;
```

Comment:
I think the `MortarSize` type should be in `dg` instead of `Spectral`, since also this tag (rightly) is. I suggest to move `Projection.*pp` to `NumericalAlgorithms/DiscontinuousGalerkin`

Reply:
I don't really see the argument here because I view projection as a spectral operation: it is a method of best-representing a function in one space in a different space. That alone is not dG specific. Having the tag inside dG I view as an application of those functions.

Reply:
I agree with the projection operation being in `Spectral`. But it bothers me a little that `MortarSize` is in `Spectral`, since that's more of a `Domain` or `dg` thing..

Reply:
Okay, I'm merging, we can rename later when we come up with a better name. Added an issue in #870, need to keep moving :)